lnme.go

```diff
@@ -166,7 +166,7 @@ func main() {
 				}
 				sats := msats / 1000 // we need sats
 				metadataHash := sha256.Sum256([]byte(lnurlMetadata))
-				invoice, err := lnClient.AddInvoice(sats, lightningAddress, metadataHash[:])
+				invoice, err := lnClient.AddInvoice(sats, lnurlMetadata, metadataHash[:])
```

Comment:
hmm. this is actually the memo/description of the invoice. Wondering if the description hash (metadataHash) is passed on correctly. how does your invoice look like?

Reply:
`lnbc20n1ps4rjkkpp5v3gz0d4d326ypxju2lqg9zy65n9cg8m2yt9vg8d350x6znglyx9qdr8vcexgctzx4jrzd34v3nx2dmyxyunsvm9x9skyvpkxf3nscmxvv6nzctyxenrxc35xd3xxe3cx9jkvwpkvf3rxwpcve3nqd3evccnzvqcqzpgxqyz5vqsp5pvd5kta0nt4m34vgpfemkzp7ls7qhh744tyra4ud7rdug4ftmtqs9qyyssqu2pypcqkcmxhdnnh73j5j5yk9k6dt0wzwahea63raduc2cqg2jy3pxel2h7c82snwk7uutqnvc5d8zw58tlu8cwrk79alrmchgzkxvqq79srhk` for instance?

Reply:
I think this problem is fixed in master with this commit: https://github.com/bumi/lnme/commit/4fb8fa9a737a228310b1f8e1b10d605681cfed6b
but this commit hasn't made it into the 1.1.0 release. v1.2.0 should be fine